book/A-git-in-other-environments/sections/atom.asc

```diff
@@ -0,0 +1,23 @@
+=== Git in Atom
+
+(((Atom)))
+Atom ships with a built-in package called GitHub, which provides a fairly-complete interface to Git operations and a deep integration with the website GitHub.
+There are different ways to access the package, probably the most common way is through the keybinding:
+
+Open the Git panel: `Ctrl+9`
+
+Another way is from the menu: `Packages -> GitHub -> Toggle Git Tab`
+
+GitHub comes with plenty of documentation when it comes to the integration with a remote repository on GitHub, which you can find within atom "Flight Manual".
+But the GitHub package can also integrate with any remote repository.
+
+=== Clone repositories
+
+To clone a repository, access the command palette by typing `Ctrl + Alt + P` and then run the `GitHub: Clone` command.
+In the dialog paste a URL of a repository and click "Clone". A new project will get added to the Tree View.
+The URL to the remote repository for SSH connection should be of the form:
+
+    ssh://git@sub.domain.com:portNB/path/project.git
+
+For this operation to succeed, make sure that your git and your ssh are properly configured.
+Git is configured from a file called `.gitconfig` which must be located in your user directory (`%USERPROFILE%` on windows, run `git config` on Linux environments). Similarly SSH must be configured in a file `/.ssh/config` also located within your user directory on windows environement. 
```

Comment:
Rationale for change:
- The word "Windows" is a noun, it refers to a specific thing, so it needs to be capitalized.
- The spelling for the word environement is wrong, it is: environment.
---
Suggested situation after implementing changes:

For this operation to succeed, make sure that your git and your ssh are properly configured.
Git is configured from a file called `.gitconfig` which must be located in your user directory (`%USERPROFILE%` on **Windows**, run `git config` on Linux environments). Similarly SSH must be configured in a file `/.ssh/config` also located within your user directory on **Windows** **environment**.